src/main/java/nl/tudelft/ti2206/group9/level/save/Writer.java

```diff
@@ -87,9 +87,15 @@ static String encrypt(final String input) {
     private static String writeToJSON() {
         final JSONObject mainObject = new JSONObject();
 
-        final JSONObject settings = new JSONObject();
-        settings.put("soundEnabled", State.isSoundEnabled());
-        mainObject.put("settings", settings);
+        final JSONObject soundtracksettings = new JSONObject();
+        soundtracksettings.put("soundtrackEnabled",
+                State.isSoundtrackEnabled());
+        mainObject.put("soundtracksettings", soundtracksettings);
+
+        final JSONObject soundeffectsettings = new JSONObject();
+        soundeffectsettings.put("soundEffectsEnabled",
+                State.isSoundEffectsEnabled());
+        mainObject.put("soundEffectssettings", soundeffectsettings);
```

Comment:
I would prefer it if you could save all settings (soundeffect and soundtrack) in the settings object, not the main object

<review